app.js

```diff
@@ -9,48 +8,163 @@ function Store(name, minCustomers, maxCustomers, averageCookiePerSale) {
+  }
+  var totalHeader = document.createElement('th');//creating a total header to append to the end of the hours row
+  totalHeader.textContent = 'Total';
+  hoursRow.appendChild(totalHeader);
+  tableEl.appendChild(hoursRow);
+
+  for(obj in cookieStores) { //for each object in cookiestores, object will represent the index (0, 1 ,2 ,3 ,4,5) in hours open
+    cookieStores[obj].cookieSalePerHour();//gets object you're iterating over
+    var tableRow = document.createElement('tr');//create new row
+    var nameRow = document.createElement('td');
+    nameRow.textContent = cookieStores[obj].name;//adds a name to the td in the nameRow
+    tableRow.appendChild(nameRow);
+
+    for (hour in timeArray) {
+      var tableData = document.createElement('td');
+      tableData.textContent = cookieStores[obj].cookiesPerHour[hour];//getting hourlySales at 10,11,12,1,2, while the cookieStore stays on the pikeplace object.
+      tableRow.appendChild(tableData);
+    }
+    var totalsData = document.createElement('td');
+    totalsData.textContent = cookieStores[obj].totalSales;
+    tableRow.appendChild(totalsData);
+    tableEl.appendChild(tableRow);//appending to original tr
+  }
 };
 
-var pikeStore = new Store('Pike Place', 17, 88, 5.2);//why an array at index 4?
+var pikeStore = new Store('Pike Place', 17, 88, 5.2);
 var seaTacStore = new Store('SeaTac', 6, 24, 1.2);
 var southCenter = new Store('Southcenter', 11, 38, 1.9);
 var belleSquare = new Store('Bellevue Square', 20, 48, 3.3);
 var alki = new Store('Alki', 3, 24, 2.6);
 
-pikeStore.render();
-seaTacStore.render();
-southCenter.render();
-belleSquare.render();
-alki.render();
+Store.render();
+
+Store.renderNew = function(obj) {
+  console.log(obj);
+  var formRow = document.createElement('tr');
+  var formNameTd = document.createElement('td');
+  formNameTd.textContent = obj.name;
+  formRow.appendChild(formNameTd);
+  for(hour in timeArray) {
+    var salesTd = document.createElement('td');
+    salesTd.textContent = obj.cookiesPerHour[hour];
+    console.log(obj.cookieSalePerHour[hour]);
+    formRow.appendChild(salesTd);
+  }
+  var newTotalTd = document.createElement('td');
+  newTotalTd.textContent = obj.totalSales;
+  formRow.appendChild(newTotalTd);
+  tableEl.appendChild(formRow);
+};
+
+var formEl = document.getElementById('form');
+
+formEl.addEventListener('submit', function(event) {//ask about bubbling
+  event.preventDefault();
+  var newStoreName = event.target.newstorelocation.value;//newstorelocation is in html
+  var newMinCust = parseInt(event.target.min.value);//id = min
+  var newMaxCust = parseInt(event.target.max.value);//id = max
+  var newAvgCustSale = parseFloat(event.target.avg.value);
+  var newShop = new Store(newStoreName, newMinCust, newMaxCust, newAvgCustSale);
+  newShop.cookieSalePerHour();
+  console.log(newShop);
+  Store.renderNew(newShop);
+});
+
+// function Store(name, minCustomers, maxCustomers, averageCookiePerSale) {
```

Comment:
Much Comment. Clean up the unnecessary comments, and it'll help keep your head clear.